crossgovernmentservices/csd_notes
# Notes owners

## What
- Refactored migrations to include `author` attribute on Note model - BREAKING
- Added SQLite specific constraint naming scheme and batch-mode migrations to
  make migrations work in tests
- Refactored tests to accommodate required author attribute when creating notes
- Refactored test fixture for logged in user, and fixed email tip tests which
  broke as a result of naive cookie handling
- Raise 404 error if user tries to edit another user's note
- Filter notes list to those authored by current user
- Refactor email tip code into separate module
## How to review
1. Run app as per README, except add 2 users to Dex's `users.json`
2. Login as the first user
3. Add some notes
4. Logout and login as the second user
5. See no notes
## Who can review

Not @andyhd
